### ISSUE TITLE
Write integration tests

### ISSUE THREAD

**Issue by anmonteiro** [anmonteiro/lumo#251]
Some features are really hard to test simply with unit tests.

We should have some integration tests that actually exercise Lumo with expected input & output

**Comment by jstaffans** [comment]
Is this issue still relevant? 

Isn't Lumo exercised as part of tests since https://github.com/anmonteiro/lumo/pull/217 ?

**Comment by anmonteiro** [comment]
@jstaffans yes, but those are still unit tests. My idea here was to run lumo against a set of known forms and diff the result. You can think of it as kind of a smoke test too. IIRC Planck has a suite of integration tests.

**Comment by anmonteiro** [comment]
If you're interested in contributing some minimal infra for this I can help you get started.

**Comment by anmonteiro** [comment]
If you're interested in contributing some minimal infra for this I can help you get started.
